app/views/posts/show.html.erb

```diff
@@ -1 +1,10 @@
-<h1> Displays a post by id </h1>
\ No newline at end of file
+<div class="one-post">
+<h1>Post #<%= @post.id %> By <%= @post.user.name %>  </h1>
+<span class= "counter-span"> 
+<strong>Comments: <%= @post.comments.count %> </strong>
+<strong>Likes: <%= @post.likes.count %> </strong>
+ </span>
+ <div class="post-text">
+    <p><%= @post.text %> </p>
+</div>
```

Comment:
- [ ] In the post show page, User should all the comments related to that post. Please add it here.